pkg/apis/nodeup/config.go

```diff
@@ -72,7 +72,7 @@ type Config struct {
 	// Hooks are for custom actions, for example on first installation.
 	Hooks [][]kops.HookSpec
 	// ContainerdConfig config holds the configuration for containerd
-	ContainerdConfig string `json:"containerdConfig,omitempty"`
+	ContainerdOverrideConfig *string `json:"containerdConfig,omitempty"`
```

Comment:
I want to add `ContainerdConfigBase` so that people can merge some custom config with what kOps builds.
```suggestion
	ContainerdConfigOverride *string `json:"containerdConfigOverride,omitempty"`
```